### DISCUSSION TITLE
Watcom toolchain wrappers

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#985] [Ideas]
To make people switch from OpenWatcom to Orange C easier.

Watcom pros: OpenWatcom supports Linux and has full support for 16 bit DOS.

Orange C pros: more standard compliant and more up-to-date headers (taken from Pelles C).

**Comment by GitMensch** [comment]
What do you miss in occ or in gccocc? Can't you use these? What do you expect from a wrapper which is not available this way?

**Reply by ghost** [reply]
To make people switch from OpenWatcom to Orange C easier.

**Reply by ghost** [reply]
But, you are right. OpenWatcom itself has wrappers for popular compilers, too. So these people could learn to use these wrappers before switching to Orange C. The wrappers for the popular compilers are available here, too. Perhaps I'm thinking too much.
